podcasts/End of Year/Stories/Views/PodcastCover.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import PocketCastsServer
+import Kingfisher
+
+struct PodcastCover: View {
+    /// UUID of the podcast to load the cover
+    let podcastUuid: String
+
+    /// Whether this is a big cover, in which shadows should be bigger
+    let big: Bool
+
+    @State private var image: UIImage?
+
+    init(podcastUuid: String, big: Bool = false) {
+        self.podcastUuid = podcastUuid
+        self.big = big
+    }
+
+    var body: some View {
+        ZStack {
+            Group {
+                if big {
+                    Rectangle()
+                        .modifier(BigCoverShadow())
+                } else {
+                    Rectangle()
+                        .modifier(NormalCoverShadow())
```

Comment:
For some reason, I can't get the shadow to work when applied to the image itself. This is a workaround around this issue.